### ISSUE TITLE
Incorrect tokenization of IRIs containing reserved words

### ISSUE THREAD

**Issue by GoogleCodeExporter** [liveontologies/elk-reasoner#5]
```
org.semanticweb.elk.owl.parsing.javacc.ParseException is thrown when NCI.owl 
(same comments apply to NCI-Disjunction.owl) given as input.

I have attributed the bug to the existence of the class IRIs:

Declaration(Class(nciOncology:Class))       line 15281 of NCI.owl
Declaration(Class(nciOncology:Ontology))    line 48495 of NCI.owl

Note that it is not the declarations that are causing the problem, since if 
those lines are deleted from the owl file, then the lines which would have 
immediately followed the deleted lines (and are the only place in the ontology 
other than the declarations which use nciOncology:Class or 
nciOncology:Ontology), then errors still persist.

In short, nciOncology:Class and nciOncology:Ontology are not permitted as IRIs, 
when they should be. The temporary fix is just to rename them to 
nciOncology:ClassXYZ and nciOncology:OntologyXYZ everywhere, and then the file 
can be parsed.
```

Original issue reported on code.google.com by `andy...@hotmail.co.uk` on 10 Oct 2011 at 7:18
